app/(api)/api/elections/route.ts

```
@@ -1,11 +1,15 @@
 import { NextResponse } from "next/server";
 import prisma from "@/prisma/prisma"; // Adjust the import path to match your project structure
 import { normalizeState } from "@/lib/stateMapping";
-import { Election } from "@prisma/client";
+
+type Election = {
```

ESLint

[error] 'Election' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)